src/redux/Action/fetchallbuses.js

```diff
@@ -1,14 +1,102 @@
 import * as actions from './actiontypes.js'
-// import Axios from './axiosConfig'
 import axios from 'axios'
-export const fetchAllbuses = () => dispatch => {
-    return axios.get(`http://localhost:5000/api/v1/buses/`)
-    .then(res => {
-      dispatch({
-        type:actions.FETCH_ALL_BUSES ,
-        payload: res.data.buses,
-
-      })
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css"
+
+const notify = (toastMsg) => toast(toastMsg);
+const baseUrl = process.env.BACKEND_URL
+
+const token = localStorage.getItem('token')
+export const fetchAllbuses = () => async (dispatch) => {
+    const res = await axios.get(`${baseUrl}/buses`,{
+      headers: {
+        "Authorization": `Bearer ${token}`
       }
-    )
+    });
+  dispatch({
+    type: actions.FETCH_ALL_BUSES,
+    payload: res.data.buses.rows,
+  });
+
+  };
+
+  export const deletebus = (id) => async (dispatch) => {
+    try{
+      const res = await axios.delete(`${baseUrl}/buses/${id}`,{
+        headers: {
+          "Authorization": `Bearer ${token}`
+        }
+      });
+      const message = res.data.message
+    dispatch({
+      type: actions.DELETE_BUS,
+      payload: {id:id,message:res.data.message},
+    });
+    notify(message)
+    }catch(err){
+      console.log(err.response.data)
```

Comment:
As long as you have implemented a toast notification pop up, hope you have remove these console log message in this file.